Open_Acidification_pH-stat_arduino/Open_Acidification_pH-stat_arduino.ino

```diff
@@ -49,20 +49,30 @@ EthernetServer ethernet_server(80);
 int request_char_counter = 0;  // for parsing html requests
 String data;                   // GET query with data
 
-float interval = 1200000;  // interval at which to update Google Sheets (milliseconds)
+// Time Constants /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+int const SECOND = 1000;
+int const MINUTE = 60 * SECOND;
+int const HOUR = 60 * MINUTE;
+int const DAY = 24 * HOUR;
+int const WEEK = 7 * DAY;
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+float interval = 20 * MINUTE;  // interval at which to update Google Sheets (milliseconds)
```

Comment:
I am curious about how these variables are being used. Should these be a float? Can these be const? (review all time or interval related times listed here.)

Reply:
The interval is changed in loop.ino so I don't think it can be a const, though it seems unreasonable for it to go into decimals in milliseconds, thus I am considering changing it to an int